backend/typescript/services/implementations/campService.ts

```diff
@@ -68,6 +66,33 @@ class CampService implements ICampService {
         return [];
       }
 
+      if (year) {
+        // Filter out camps that have camp sessions in the desired year
+        /* eslint-disable no-param-reassign */
+        camps = camps.filter((camp) => {
+          // Go through every camp session in a camp. If it has the desired year, include it in the results.
+          /* eslint-disable no-param-reassign */
+          camp.campSessions = (camp.campSessions as CampSession[]).filter(
+            (campSession) => {
+              return campSession.dates.every((campDate) => {
```

Comment:
nit: Is it any easier to sort camp sessions by dates and check the first + last date? For the size that we're dealing with, it's not a huge performance thing but might be easier to read

Reply:
For the sake of time will just move on with the current code since it works but will keep this in mind in case it does become an issue in the future!